frontend/src/shared/serverUrls.js

```diff
@@ -21,7 +21,12 @@ export const DURATIONS_LASTWEEK_URL = "/api/minutes_lastweek";
 export const DURATIONS_TOTAL_URL = "/api/minutes_total";
 
 // personalanalysisview API
+export const PERSONAL_DURATION_AVG_URL = "/api/personal_duration_avg";
 export const PERSONAL_INTERVALS_URL = "/api/personal_intervals";
 export const PERSONAL_DURATIONS_URL = "/api/personal_durations";
-export const PERSONAL_INTERVALS_PER_WEEK_URL = "/api/personal_intervals_per_week";
-export const SERVER_BASE_URL = process.env.NODE_ENV === "development" ? SERVER_BASE_URL_DEV : SERVER_BASE_URL_PROD;
\ No newline at end of file
+export const PERSONAL_INTERVALS_PER_WEEK_URL =
```

Comment:
- [ ] If this is done by Prettier, we might need to match our Prettier settings again, otherwise is OK.